packages/cli/src/collect/collect.js

```diff
@@ -95,17 +100,20 @@ async function startServerAndDetermineUrls(options) {
   if (!options.staticDistDir) {
     let close = async () => undefined;
     if (options.startServerCommand) {
+      const regexPattern = new RegExp(options.startServerReadyPattern, 'i');
```

Comment:
We should honour the default regex `/(listen|ready)/i` if no specific regex is provided. Am I missing anything?

Reply:
yep it's specified as the default value in yargs config above @akanshgulati

Reply:
The default regex is set as the `startServerReadyPattern` default.